tgui/packages/tgui/interfaces/SecureSafe.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author blackrep (https://github.com/blackrep)
+ * @license MIT
+ */
+
+
+import { useBackend } from '../backend';
+import { Stack, Box, ProgressBar, Section, Flex, Button, Blink } from '../components';
+import { Window } from '../layouts';
+
+import { capitalize, glitch } from './common/stringUtils';
+
+const KEY_PAD_INPUT_LAYOUT = [
+  ['7', '8', '9', 'A'],
+  ['4', '5', '6', 'B'],
+  ['1', '2', '3', 'C'],
+  ['0', 'F', 'E', 'D'],
+];
+
+const stylizeCode = (attempt, codeLen) => {
+  return attempt.padEnd(codeLen, "*").split("").join(" ");
+};
+
+const SecureSafeScreen = (props, _context) => {
+  const {
+    attempt,
+    codeLen,
+    disabled,
+    emagged,
+    padMsg,
+  } = props;
+
+  let content = padMsg ? padMsg : stylizeCode(attempt, codeLen);
+  if (disabled) {
+    content = "NO ACCESS";
+  }
+  if (emagged) {
+    content = glitch(content, 2);
+  }
+  return (
+    <Box
+      fontSize="25px"
+      fontFamily="Courier"
+      bold
+      textAlign="center"
+      style={{
+        "padding": "3px",
+        "background-color": "#342210",
+        "border-width": "0.1em",
+        "border-style": "solid",
+        "border-radius": "0.16em",
+        "border-color": "#FC8E1F",
```

Comment:
can't this be done in the jsx

Reply:
padding, backgroundColor can be done in JSX, which I'll correct tomorrow. However border props aren't supported according to the component reference:
https://github.com/goonstation/goonstation/blob/master/tgui/docs/component-reference.md#box

Reply:
ahh you're right yeah i meant as props

Reply:
Put `padding` and `backgroundColor` into `Box` props. The rest still require the style object since they aren't supported.